### ISSUE TITLE
Could someone explain what this does in layman's terms?

### ISSUE THREAD

**Issue by zwhitchcox** [wearefractal/vinyl#59]
I've read the docs and that article twice, but I still don't really understand what this does and how it does it. I get the fact that it somehow adds information about the file and the path to the stream, but that's about everything I know, and it seems like it does more than that

**Comment by contra** [comment]
This is an object for containing meta-data about files.
